gulpfile.js

```diff
@@ -66,6 +66,7 @@ let settings = {
     },
     browserslist: ["> 2%", "last 2 versions", "IE 11", "not dead"],
     sassSourcemaps: true,
+    quietSassDeps: true
```

Comment:
**suggestion:** Wondering if we can improve clarity with the setting name here to highlight that this is about Sass "warnings", not just "deps" (This also clarifies that we are talking about deprecations, not dependencies). 

Also wondering if it should be written in the negative, so that we can set  "false" as a default. This part is very much a quibble and could just be personal preference, but  it felt a bit odd to add a `settings.compile.quietSassDeps = false` to the gulpfile. 

Curious what you think!

```suggestion
    sassDeprecationWarnings: false
```

Reply:
Hmm I'm divided here. While I do think that the naming convention your suggesting makes sense, part of me thinks we should match what Sass is providing. Since [quietDeps](https://sass-lang.com/documentation/js-api/interfaces/options/#:~:text=Optional%20quietDeps&text=If%20this%20option%20is%20set,entrypoint%20are%20not%20considered%20dependencies.) is the name of setting I see potential value in keeping ours close to it.

Do you see any value in changing it to `sassQuietDeps` so we're sort of "prefixing" the Sass setting name with "Sass"?

Alternatively, we use this approach which would probably be more readable for users unfamiliar with this Sass deprecation 🤔 

```js
sassDeprecationWarnings: false
...
sass({
  ...
  quietDeps: !settings.compile.sassDeprecationWarnings,
})
```

Reply:
Renamed in https://github.com/uswds/uswds-compile/pull/131/commits/7a4c4347150067dab65a3a4fd012c83b8a7e0df9!